packages/toolpad-components/src/Text.tsx

```diff
@@ -8,9 +8,8 @@ import {
   styled,
 } from '@mui/material';
 import { createComponent } from '@mui/toolpad-core';
-import remarkGfm from 'remark-gfm';
 
-const ReactMarkdown = React.lazy(() => import('react-markdown'));
+const ReactMarkdown = React.lazy(() => import('markdown-to-jsx'));
```

Comment:
```suggestion
const Markdown = React.lazy(() => import('markdown-to-jsx'));
```
Let's rename to just `Markdown` now that we don't use `react-markdown` anymore?